model/riscv_insts_cdext.sail

```diff
@@ -14,6 +14,7 @@
  */
 
 /* ****************************************************************** */
+$[name "compressed floating-point load double-precision from stack pointer"]
```

Comment:
Let's put the action word first: load
Then the target: double-precision floating-point
Then the action modifier: from stack pointer
Then the "class": (compressed)
So, "load double-precision floating point from stack pointer (compressed)".

Similarly for the others in this commit, please. Thanks!